Dockerfile.in

```diff
@@ -3,15 +3,23 @@ FROM DISTRO
 ARG SOURCE_DIR=/ffmpeg
 ARG ARTIFACT_DIR=/dist
 # Docker run environment
+ENV DEB_BUILD_OPTIONS=noddebs
+ENV DEBIAN_FRONTEND=noninteractive
 ENV ARCH=BUILD_ARCHITECTURE
 ENV GCC_VER=GCC_RELEASE_VERSION
 ENV SOURCE_DIR=/ffmpeg
 ENV ARTIFACT_DIR=/dist
-ENV DEB_BUILD_OPTIONS=noddebs
+ENV TARGET_DIR=/usr/lib/jellyfin-ffmpeg
+ENV PKG_CONFIG_PATH=${TARGET_DIR}/lib/pkgconfig:$PKG_CONFIG_PATH
+ENV LD_LIBRARY_PATH=${TARGET_DIR}/lib:${TARGET_DIR}/lib/mfx:${TARGET_DIR}/lib/xorg:${LD_LIBRARY_PATH}
```

Comment:
why is this needed?

Reply:
Since the installation location of the libva/qsv header and pkgconfig is modified, we need to manually specify their paths so that they can be found when configure ffmpeg.